jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/Shared.java

```diff
@@ -75,15 +76,24 @@ public static SharedClassHelperFactory getSharedClassHelperFactory() {
 
 	/**
 	 * If sharing is enabled, returns a SharedDataHelperFactory, otherwise returns null.
-	 * <p>
-	 * @return		SharedDataHelperFactory
+	 *
+	 * @return SharedDataHelperFactory
 	 */
 	public static SharedDataHelperFactory getSharedDataHelperFactory() {
-		synchronized(monitor) {
-			if (shdHelperFactory==null && isSharingEnabled()) {
+		synchronized (monitor) {
+			if (shdHelperFactory == null && isSharingEnabled()) {
 				shdHelperFactory = new SharedDataHelperFactoryImpl();
 			}
 			return shdHelperFactory;
 		}
 	}
+
+	/**
+	 * Constructs a new instance of this class.
+	 */
+	@Deprecated
```

Comment:
For newly added constructors, I see `@Deprecated` here only, not in other files.

Reply:
As this class defines only `static` features, I don't think it was ever intended to be constructed. Removing the implicit public constructor has the potential to break existing client code, hence the annotation.

Reply:
Might be worth adding a comment in the code explaining the annotation ?

Reply:
Comment added in 8f40feddf477e84b33f8c6376b331082f9fc8cf7.